libcudacxx/include/cuda/std/detail/libcxx/include/cmath

```diff
@@ -678,7 +678,7 @@ __constexpr_isfinite(_A1 __lcpp_x) noexcept
     return isfinite(__lcpp_x);
 }
 
-#if defined(_MSC_VER) || defined(__CUDACC_RTC__) || defined(_LIBCUDACXX_COMPILER_CLANG_CUDA)
+#if defined(_MSC_VER) || defined(__CUDACC_RTC__) || defined(_CCCL_CUDA_COMPILER_CLANG)
```

Comment:
We should clean this up also:
```suggestion
#if defined(_CCCL_COMPILER_MSVC) || defined(_CCCL_COMPILER_NVRTC) || defined(_CCCL_CUDA_COMPILER_CLANG)
```

Applies below

Reply:
Do you mean in all other files?
It might be better to make a separate PR for this, whats your opinion?

Reply:
Oh only when we already change the line